backend/app/core/db.py

```
@@ -1 +1,194 @@
+        try:
+            async with self.acquire() as conn:
+                # 外部キー制約を一時的に無効化
+                await conn.execute("SET session_replication_role = 'replica';")
+                
+                # 全テーブル削除
+                tables = await conn.fetch(
+                    "SELECT tablename FROM pg_tables WHERE schemaname = 'public';"
+                )
+                
+                for table in tables:
+                    await conn.execute(f"DROP TABLE IF EXISTS {table['tablename']} CASCADE;")
```

Flake8 (7.2.0)

[error] line too long (93 > 79 characters)

(E501)